src/Sulu/Bundle/PreviewBundle/Preview/Renderer/PreviewRenderer.php

```diff
@@ -155,7 +155,7 @@ public function render($object, $id, $webspaceKey, $locale, $partial = false)
         $this->eventDispatcher->dispatch(Events::PRE_RENDER, new PreRenderEvent($attributes));
 
         try {
-            $response = $this->httpKernel->handle($request, HttpKernelInterface::SUB_REQUEST, false);
+            $response = $this->httpKernel->handle($request, HttpKernelInterface::MASTER_REQUEST, false);
```

Comment:
to be sure that all listener are called the request has to be rendered as master-request. this makes currently problems with the websocket. i have to check this tomorrow.